srcdeps-core/src/main/java/org/srcdeps/core/config/scalar/Scalars.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright 2015-2016 Maven Source Dependencies
+ * Plugin contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.srcdeps.core.config.scalar;
+
+import java.lang.reflect.Type;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.srcdeps.core.BuildRequest.Verbosity;
+import org.srcdeps.core.config.tree.ScalarDeserializer;
+
+/**
+ * Hosts scalar types related helper methods.
+ *
+ * @author <a href="https://github.com/ppalaga">Peter Palaga</a>
+ */
+public final class Scalars {
+
+    /**
+     * {@link Map} from types we consider primitive to their respective {@link ScalarDeserializer}s.
+     */
+    private static final Map<Type, ScalarDeserializer> SCALAR_TYPES;
+
+    static {
+
+        /* Put a ScalarDeserializer for every scalar type to scalarTypes map */
+        Map<Type, ScalarDeserializer> primitives = new HashMap<>();
+        primitives.put(String.class, new ScalarDeserializer() {
+            @Override
```

Comment:
Hehe. At one hand, I'd gladly kick Java 7 out of here, but OTOH, being able to build EAP 6 as a srcdep is such a huuuge advantage for me, that I am quite OK to suffer without lambdas here :).